### PR TITLE
begin adding arturom-uci bio

### PR DESCRIPTION
**Pull request by arturom-uci**
:boom: Still in progress:exclamation: 

This work is related to #51

### CODE REVIEW

File: docs/arturom-uci [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# GitHub Student
+- Name: Arturo Martinez
```
Comment (arturom-uci) [inline docs/arturom-uci]: Need more info asap!!

**Comment by arturom-uci** [comment]
will be adding more work soon.... :+1:

**Comment by arturom-uci** [comment]
@githubteacher please add this change to master. :smile:

**Comment by arturom-uci** [comment]
@grodrigu can you review my work.

**Comment by grodrigu** [comment]
:+1: go ahead and commit.

#unauthorizedAuthorization

**Pull request merged**